### PR TITLE
Add undocumented tournament Config options

### PR DESCRIPTION
**Pull request by panpawn**
These are used within the tournament code, but they aren't documented...

### CODE REVIEW

**Comment by Zarel** [comment]
...why is there a camelcase option?

Can you change it to fit the rest of the options?

**Comment by panpawn** [comment]
Yea, sure

**Comment by panpawn** [comment]
Done

File: config/config-example.js [hunk 1]
```diff
@@ -179,8 +179,12 @@ exports.customavatars = {
 // tourroom - specify a room to receive tournament announcements (defaults to
 // the room 'tournaments').
 // tourannouncements - announcements are only allowed in these rooms
+// tournamentdefaultplayercap - a set cap of how many players can be in a tournament
+// istournamentsrated - toggles tournaments being ladder rated (true) or not (false)
 exports.tourroom = '';
 exports.tourannouncements = [/* roomids */];
+exports.tournamentdefaultplayercap = false;
+exports.istournamentsrated = false;
```
Comment (Zarel) [inline config/config-example.js]: To make these consistent with the others, I think I'd prefer `tourdefaultplayercap` and `ratedtours`

Comment (panpawn) [inline config/config-example.js]: done

**Comment by panpawn** [comment]
Updated

**Comment by panpawn** [comment]
Was there something else that I should do with this?

**Pull request merged**